### PR TITLE
Reactjs

### PR DESCRIPTION
**Pull request by bsshashank**
Hi Andreas,

I've implemented the SettingsManager and updated the PluginsManager.

### CODE REVIEW

File: app/components/PluginCard.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
...operties(Constructor.prototype,protoProps);if(staticProps)defineProperties(Constructor,staticProps);return Constructor}}();var _react=require('react');var _react2=_interopRequireDefault(_react);var _radium=require('radium');var _radium2=_interopRequireDefault(_radium);var _reactMdl=require('react-mdl');function _interopRequireDefault(obj){return obj&&obj.__esModule?obj:{default:obj}}function _classCallCheck(instance,Constructor){if(!(instance instanceof Constructor)){throw new TypeError('Cannot call a class as a function')}}function _possibleConstructorReturn(self,call){if(!self){throw new ReferenceError('this hasn\'t been initialised - super() hasn\'t been called')}return call&&(typeof call==='object'||typeof call==='function')?call:self}function _inherits(subClass,superClass){if(typeof superClass!=='function'&&superClass!==null){throw new TypeError('Super expression must either be null or a function, not '+typeof superClass)}subClass.prototype=Object.create(superClass&&superClass.prototype,{constructor:{value:subClass,enumerable:false,writable:true,configurable:true}});if(superClass)Object.setPrototypeOf?Object.setPrototypeOf(subClass,superClass):subClass.__proto__=superClass}/**
```
Comment (appelgriebsch) [inline app/components/PluginCard.js:1]: Pls. do not check-in generated code... The .jsx files are sufficient!

File: app/main.html [hunk 2]
```diff
@@ -21,6 +21,6 @@
     <div role="app"></div>
 
     <script type="application/javascript" src="node_modules/react-mdl/extra/material.min.js"></script>
-    <script type="application/javascript" src="main.js"></script>
+    <script type="application/javascript" src="app.js"></script>
```
Comment (appelgriebsch) [inline app/main.html:24]: Is main.js no longer needed? Why not changing main.js instead of introducing app.js(x)?

**Review by appelgriebsch — COMMENTED** [review]
Please also remove the .asar files from plugins/ directory. We should discuss providing some ``example´´ projects for them...

File: app/main.html (hunk 2 above)
Comment (bsshashank) [reply app/main.html:24]: Sorry, I forgot to integrate it with main.js. I didn't want to change main.js directly. I'll make the changes.

File: app/components/PluginCard.js (hunk 1 above)
Comment (bsshashank) [reply app/components/PluginCard.js:1]: Okay, I'll add it into the .gitignore file

**Pull request merged**